### ISSUE TITLE
Container images are not removed in `setup.sh --completely-remove`

### ISSUE THREAD

**Issue by jppaulo06** [kworkflow/kworkflow#1017]
**Description:**

After completely removing kw, images from integration tests are not removed.

**How to Reproduce:**

1. With unstable kw installed, execute in kw source tree
```bash
./run_tests --integration
```
2. After it finishes, you can see that the images are installed runnning
```bash
podman image ls
```
3. Then, completely remove kw with
```bash
./setup.sh --completely-remove
```
4. Run this command again to confirm that the images are still there:
 ```bash
podman image ls
```
You should see something like
```
$ podman image ls                                                                                                                  [19:43:36]
REPOSITORY                   TAG         IMAGE ID      CREATED       SIZE
localhost/kw-fedora          latest      eef06db6345a  6 hours ago   1.16 GB
localhost/kw-debian          latest      40227dd4c41a  6 hours ago   2.39 GB
localhost/kw-archlinux       latest      e912e0d8d8dd  6 hours ago   1.24 GB
docker.io/library/debian     latest      a6916e41aa87  20 hours ago  121 MB
docker.io/library/archlinux  latest      69f38d8f6347  10 days ago   446 MB
docker.io/library/fedora     latest      8404925a71fd  6 weeks ago   183 MB
```

**Expected behavior:**

It is expected that the images are gone after completely removing kw from system.

**System information:**

 - OS: Linux fedora 6.6.9
 - kw Version: 
``` 
alpha
Branch: unstable
Commit: 1764e42
```
 - Bash Version 5.2.21

**Comment by uwla** [comment]
I'd like to share my view here.

I think it makes sense that the container images are not removed when using the `setup.sh --completely-remove` flag because podman is not installed and the images are not built locally when running `setup.sh --install` since podman is a dev dependency and the images will only be build if the user is a developer, not and end-user.

There is a related issue, issue #997, to add a `--install-dev` dependency, which would install shunit, shellcheck, podman, among other dependencies required to run tests and code linter. It would make sense to have a `setup.sh --uninstall-dev` flag to uninstall dev dependencies, but I also think it should not attempt to clear the podman images. My proposed solution is that the `run_tests.sh` script should have a `--clear-cache` flag to clear all cache related to kw tests.

To sum up: running `setup --completely-remove` has the correct behavior currently, and we should add a `--clear-cache` to `run_tests.sh` to clear the podman images.

**Comment by jppaulo06** [comment]
> I'd like to share my view here.
> 
> I think it makes sense that the container images are not removed when using the `setup.sh --completely-remove` flag because podman is not installed and the images are not built locally when running `setup.sh --install` since podman is a dev dependency and the images will only be build if the user is a developer, not and end-user.
> 
> There is a related issue, issue #997, to add a `--install-dev` dependency, which would install shunit, shellcheck, podman, among other dependencies required to run tests and code linter. It would make sense to have a `setup.sh --uninstall-dev` flag to uninstall dev dependencies, but I also think it should not attempt to clear the podman images. My proposed solution is that the `run_tests.sh` script should have a `--clear-cache` flag to clear all cache related to kw tests.
> 
> To sum up: running `setup --completely-remove` has the correct behavior currently, and we should add a `--clear-cache` to `run_tests.sh` to clear the podman images.

Didn't think of this before. I agree with you; liked the solution as well :)
